### PR TITLE
+ Implement a basic modal

### PR DESCRIPTION
**Pull request by JPLachance**

### CODE REVIEW

File: docs/_includes/scripts.html [hunk 1]
```diff
@@ -26,5 +26,35 @@
             $(this).addClass('state-active');
             selectedTab = $(this);
         });
+
+        // Simple script to handle opening/closing modals
+        function modalHandler() {
+            var backdrop = $('.modal-backdrop');
+
+            [].slice.call($('.js-modal-trigger')).forEach(function(modalTrigger) {
```
Comment (wfortin) [inline docs/_includes/scripts.html]: $('.js-modal-trigger').each()

image

File: docs/_includes/scripts.html [hunk 2]
```diff
@@ -26,5 +26,35 @@
...
+                var modal = $('#' + modalTrigger.getAttribute('data-modal'));
+                var closeButton = modal.find('.js-modal-close');
+
+                function removeModal() {
+                    modal.removeClass('opened');
+                    backdrop.addClass('closed');
+                }
+
+                modalTrigger.addEventListener('click', function() {
```
Comment (wfortin) [inline docs/_includes/scripts.html]: modalTrigger.on('click',

File: scss/components/modal.scss [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+
+.modal-container {
+  align-items: center;
+  justify-content: center;
+  z-index: $base-modal-z-index;
+
+  display: flex;
+
+  visibility: hidden;
+  pointer-events: none;
```
Comment (wfortin) [inline scss/components/modal.scss]: Might want to add a comment that this is needed to support the close modal on click outside.

File: scss/components/modal.scss [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+      color: $pure-white;
+
+      background-color: $dark-blue;
+
+      h1 {
+        overflow: hidden;
+
+        line-height: 30px;
+        text-overflow: ellipsis;
+        white-space: nowrap;
```
Comment (wfortin) [inline scss/components/modal.scss]: Do we really want the title not to wrap ?

**Comment by wfortin** [comment]
Small fixes here and there, other than that :shipit:

File: docs/_includes/scripts.html (hunk 2 above)
Comment (GermainBergeron) [inline docs/_includes/scripts.html]: `modalTrigger.click(function() {...`

**Comment by GermainBergeron** [comment]
Apply wfortin's comments then :package:

**Comment by pochretien** [comment]
:GG: , :sheep: :it:

File: scss/components/modal.scss (hunk 4 above)
Comment (JPLachance) [inline scss/components/modal.scss]: This is a question for the design team, but it seem's legit to me.

File: docs/_includes/scripts.html (hunk 1 above)
Comment (JPLachance) [inline docs/_includes/scripts.html]: http://thecodinglove.com/post/139800177607/when-i-copy-a-script-which-seems-to-work

File: docs/_includes/scripts.html (hunk 2 above)
Comment (JPLachance) [inline docs/_includes/scripts.html]: Done.

File: scss/components/modal.scss (hunk 3 above)
Comment (JPLachance) [inline scss/components/modal.scss]: Done, I also added a "Known issue: wont work IE 10." warning.

**Pull request merged**